<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="149627" started-at="2020-07-23T06:35:21Z" finished-at="2020-07-23T06:37:50Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="149627" started-at="2020-07-23T06:35:21Z" finished-at="2020-07-23T06:37:50Z">
      <class name="com.appium.framework.AppiumJavaFramework.FirstTest">
        <test-method status="PASS" signature="startServer()[pri:0, instance:com.appium.framework.AppiumJavaFramework.FirstTest@6ddf90b0]" name="startServer" is-config="true" duration-ms="11123" started-at="2020-07-23T06:36:41Z" finished-at="2020-07-23T06:36:52Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startServer -->
        <test-method status="PASS" signature="firstTestDemo()[pri:0, instance:com.appium.framework.AppiumJavaFramework.FirstTest@6ddf90b0]" name="firstTestDemo" duration-ms="25349" started-at="2020-07-23T06:36:52Z" finished-at="2020-07-23T06:37:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- firstTestDemo -->
        <test-method status="PASS" signature="stopServer()[pri:0, instance:com.appium.framework.AppiumJavaFramework.FirstTest@6ddf90b0]" name="stopServer" is-config="true" duration-ms="52" started-at="2020-07-23T06:37:17Z" finished-at="2020-07-23T06:37:17Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- stopServer -->
      </class> <!-- com.appium.framework.AppiumJavaFramework.FirstTest -->
      <class name="com.appium.framework.AppiumJavaFramework.FailedTest">
        <test-method status="PASS" signature="startServer()[pri:0, instance:com.appium.framework.AppiumJavaFramework.FailedTest@57536d79]" name="startServer" is-config="true" duration-ms="12507" started-at="2020-07-23T06:37:17Z" finished-at="2020-07-23T06:37:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startServer -->
        <test-method status="FAIL" signature="firstTestDemo()[pri:0, instance:com.appium.framework.AppiumJavaFramework.FailedTest@57536d79]" name="firstTestDemo" duration-ms="20438" started-at="2020-07-23T06:37:30Z" finished-at="2020-07-23T06:37:50Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Key/value is not present] but found [Preference]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Key/value is not present] but found [Preference]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.appium.framework.AppiumJavaFramework.FailedTest.firstTestDemo(FailedTest.java:23)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- firstTestDemo -->
        <test-method status="PASS" signature="stopServer()[pri:0, instance:com.appium.framework.AppiumJavaFramework.FailedTest@57536d79]" name="stopServer" is-config="true" duration-ms="52" started-at="2020-07-23T06:37:50Z" finished-at="2020-07-23T06:37:50Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- stopServer -->
      </class> <!-- com.appium.framework.AppiumJavaFramework.FailedTest -->
      <class name="com.appium.framework.AppiumJavaFramework.ContextSwitchDemo">
        <test-method status="PASS" signature="startServer()[pri:0, instance:com.appium.framework.AppiumJavaFramework.ContextSwitchDemo@2b2948e2]" name="startServer" is-config="true" duration-ms="9503" started-at="2020-07-23T06:35:21Z" finished-at="2020-07-23T06:35:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- startServer -->
        <test-method status="PASS" signature="totalValidation()[pri:0, instance:com.appium.framework.AppiumJavaFramework.ContextSwitchDemo@2b2948e2]" name="totalValidation" duration-ms="70106" started-at="2020-07-23T06:35:30Z" finished-at="2020-07-23T06:36:40Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- totalValidation -->
        <test-method status="PASS" signature="stopServer()[pri:0, instance:com.appium.framework.AppiumJavaFramework.ContextSwitchDemo@2b2948e2]" name="stopServer" is-config="true" duration-ms="52" started-at="2020-07-23T06:36:40Z" finished-at="2020-07-23T06:36:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- stopServer -->
      </class> <!-- com.appium.framework.AppiumJavaFramework.ContextSwitchDemo -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
